app/src/androidTest/java/bacon/john/bitcoinpricewatch/ApplicationTest.java

```diff
@@ -0,0 +1,13 @@
+package bacon.john.bitcoinpricewatch;
+
+import android.app.Application;
+import android.test.ApplicationTestCase;
+
+/**
+ * <a href="http://d.android.com/tools/testing/testing_android.html">Testing Fundamentals</a>
+ */
+public class ApplicationTest extends ApplicationTestCase<Application> {
```

Comment:
stub from Android Studio when I made the project, kept it since we would want unit tests at some point